src/gentropy/dataset/study_locus.py

```diff
@@ -574,6 +574,41 @@ def clump(self: StudyLocus) -> StudyLocus:
         )
         return self
 
+    def exclude_region(
+        self: StudyLocus, region: str, exclude_overlap: bool = False
+    ) -> StudyLocus:
+        """Exclude a region from the StudyLocus dataset.
+
+        Args:
+            region (str): region given in "chr##:#####-####" format
+            exclude_overlap (bool): If True, excludes StudyLocus windows with any overlap with the region.
+
+        Returns:
+            StudyLocus: filtered StudyLocus object.
+        """
+        (chromosome, start_position, end_position) = parse_region(region)
+        if exclude_overlap:
+            filter_condition = ~(
+                (f.col("chromosome") == chromosome)
+                & (
+                    (f.col("locusStart") <= end_position)
+                    & (f.col("locusEnd") >= start_position)
```

Comment:
`locusStart` and `locusEnd` columns are nullable. How does this condition works if they are null?

Reply:
Just checked - it drops any rows where locusStart/locusEnd are null. `exclude_region()` also drops rows with null for the position column.